smac/tae/base.py

```diff
@@ -227,11 +227,11 @@ def run_wrapper(
                 budget=run_info.budget,
                 instance_specific=run_info.instance_specific
             )
-        except Exception:
+        except Exception as e:
             status = StatusType.CRASHED
             cost = self.cost_for_crash
             runtime = time.time() - start
-            additional_info = {}
+            additional_info = {'exception_msg': str(e)}
```

Comment:
What do you think about adding the actual traceback as [here](https://github.com/automl/auto-sklearn/blob/70d1a301fc0f5e7524c4d37ce1a2ff6ce45d8d73/autosklearn/evaluation/__init__.py#L35) to increase the amount of information for debugging?

Reply:
Thank you very much, this makes total sense to make debugging easier. I added these changes.